java/test/jmri/StringIOTest.java

```diff
@@ -0,0 +1,72 @@
+package jmri;
+
+import jmri.implementation.AbstractNamedBean;
+import org.junit.*;
+
+/**
+ * Tests for the Light class
```

Comment:
Maybe a cut and paste that this refers to Light?

Reply:
Yes. Thanks!